app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -161,7 +161,7 @@ class AamvaStateMaintenanceWindow
       ],
       'UT' => [
         # 3rd Sunday of every month 1:00 am. to 9:00 am. ET
-        { cron: '0 1 0 0 Sun#3', duration_minutes: 8 * 60 },
+        { cron: '0 1 * * Sun#3', duration_minutes: 8 * 60 },
       ],
       'VA' => [
```

Comment:
I liked the suggestion [here](https://github.com/18F/identity-idp/pull/11751#discussion_r1915337353) to use Data/Struct to enforce some of the shape of the data. Is that worthwhile?

Reply:
My only concern¹ is that it turns into a slippery slope² where it sort of makes sense as a whole CronMaintenanceWindowDuration class.

Let me have a go at implementing this as a lightweight Struct.

¹ But not a reason to not do it.
² I pretty universally reject "slippery slope" arguments. The person making them doesn't even attempt to argue that the thing at present is bad, just that in some undefined future situation, it could be bad.

Reply:
Today all the things have the exact same keys, seems low risk to making a `Data` struct for them IMO!

Reply:
Yeah, that's what I was imagining. Something along the lines of:

```ruby
    Window = Data.define(:cron, :duration_minutes)
    MAINTENANCE_WINDOWS = {
      'AL' => [
        # First Monday of each month from 1 am – 7 am ET
        Window.new(cron: '0 1 * * Mon#1', duration_minutes: 6 * 60),
      ],
      # ...
    }
```

Reply:
Beautiful! I was unaware of the behavior of the "new" Data class. You're right; it's exactly what we want. It validates the key names, and unlike a Struct also requires that both are present.

I think this slightly expands the scope of this PR from a simple test addition, but given that this is in response to something that required rolling back a release, the extra scrutiny required is not a bad thing IMHO.

Reply:
Yep, I think this is an easier suggestion in hindsight, but hopefully expanding usage helps raise awareness of it